src/components/taskDetails/ProgressUpdateCard/ProgressUpdateCardPresentation.tsx

```diff
@@ -59,33 +68,89 @@ export default function ProgressUpdateCardPresentation({
                     <h3 className={styles['progress-update-card__title']}>
                         {titleToShow}
                     </h3>
-                    <Tooltip
-                        tooltipPosition={{ top: '-25px', right: '-2rem' }}
-                        content={tooltipString}
+                    <div
+                        className={
+                            styles['progress-update-card__details-container']
+                        }
                     >
-                        <span
-                            className={
-                                styles['progress-update-card__date-container']
+                        <Tooltip
+                            tooltipPosition={
+                                isDevMode
```

Comment:
Why do you define the CSS here?

Reply:
I am passing the css as props as to set the Tooltip position. We are already doing this in the repo with tooltip component.